lib/LoginStrategy.ts

```diff
@@ -251,23 +251,41 @@ export class LoginStrategy extends BaseStrategy {
   async offchainClaimsOf(did: string): Promise<OffchainClaim[]> {
     did = this.didUnification(did);
 
-    const transformClaim = (claim: OffchainClaim): OffchainClaim => {
+    const transformClaim = (
+      claim: OffchainClaim
+    ): OffchainClaim | undefined => {
       const transformedClaim = { ...claim };
-      const didFormatFields = ['iss', 'sub', 'subject', 'did'];
+      const didFormatFields = ['iss', 'sub', 'subject', 'did', 'signer'];
+
+      let isMaliciousClaim = false;
```

Comment:
I think I would be more specific with the error. I don't think it is necessarily malicious.
```suggestion
      let invalidDIDProperty = false;
```